frontend/src/UI/Layout.module.css

```diff
@@ -1,4 +1,15 @@
 .main {
-  margin: 5rem auto;
+  margin: 0 auto;
+  padding-top: 5rem;
+  padding-bottom: 5rem;
   width: 90%;
+  min-height: calc(100vh - 62.19px - 59.98px);
```

Comment:
Instead of using calculations like this. Its better to have more dynamic way to set height based on content. Cause imagine this scenario you have to add new content to the header and footer. So you have to do manual calculation agen then replace px based on the new height right. There is a cleaner way of doing it where you don't need to recalculate the height of pixels again.

Here i've provided sample implementation. You can use this as your guide and experiment on it as you like.
https://codepen.io/kimjo77/pen/yLzeWxg

Reply:
Noted on this sir